### DISCUSSION TITLE
Path error

### DISCUSSION THREAD

**Discussion by Junaid-Q** [Lora-net/LoRaMac-node#1135] [Q&A]
Hi, there is an error while entering this command to the terminal. Can someone please help me with what should I add here?

`-DCMAKE_TOOLCHAIN_FILE="../cmake/toolchain-arm-none-eabi.cmake" \`

**Comment by meakash** [comment]
This is something that works for me on windows system.   Please read the following https://github.com/Lora-net/LoRaMac-node/discussions/1133  this will help you to set it up for windows.

![FRI-23-1](https://user-images.githubusercontent.com/48630822/127856746-222df949-3aea-4224-88d6-1104c7925f3e.png)

Does this help you?

**Reply by Junaid-Q** [reply]
Hi Akash, thank you so much. Can we please schedule a Skype meeting, I really need your help. Please let me know if possible for you. Thank you

**Reply by meakash** [reply]
I have sent you a reply back on twitter. Please check. Do you use the Windows system? other than windows, i am not sure how much help I will be able to be.

**Reply by Junaid-Q** [reply]
Hi Akash, thanks for your response. I already sent a message to your account and please let me know we can have a call if you are free. I just need a little push in setting up the LoRaWAN network. Thanks

**Comment by Junaid-Q** [comment]
Hi, how to fix this error, please?

![image](https://user-images.githubusercontent.com/12583457/128929457-53867e59-624b-410f-9408-e32ed8475f26.png)

**Reply by shabaz-git** [reply]
Hi,
It seems you are missing the characters at the end of the command line options:
`-DUSE_RADIO_DEBUG="ON" ..`
Those are required.

**Reply by Junaid-Q** [reply]
Thanks for your help. I added the two dots at the end and now it shows this error. 

![image](https://user-images.githubusercontent.com/12583457/129204524-925a1d0f-655b-4b29-a34d-71a01f6a61ae.png)
